pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/AnswerDao.java

```diff
@@ -325,6 +320,16 @@ public void accumulate(Map<Long, Answer> container, RowView view) {
             reduce(container, view);
         }
 
+        @Override
+        public Stream<Answer> stream(Map<Long, Answer> container) {
+            if (!childAnswers.isEmpty()) {
+                // There are child answers that has not been consumed by a parent answer,
+                // very likely we're querying for the child answers themselves so return them.
+                container.putAll(childAnswers);
```

Comment:
Found an issue where we were not able to fetch the individual child answers separately from the parent. This should fix it.